src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageMetadataControl.xaml.cs

```diff
@@ -69,5 +69,24 @@ private void PackageMetadataControl_DataContextChanged(object sender, Dependency
                 Visibility = Visibility.Collapsed;
             }
         }
+
+        // capture each item as it is selected, so we can unselect when treeview lostfocus
+        private void OnItemSelected(object sender, RoutedEventArgs e)
+        {
+            _dependencies.Tag = e.OriginalSource;
+        }
+
+        private void TreeView_LostFocus(object sender, RoutedEventArgs e)
+        {
+            // hide focus highlight when treeview lostfocus
+            if (_dependencies.SelectedItem != null)
+            {
+                TreeViewItem selectedTVI = _dependencies.Tag as TreeViewItem;
+                if (selectedTVI != null)
```

Comment:
Casting and checking in one line:

```suggestion
                if (_dependencies.Tag is TreeViewItem selectedTVI)
```

Reply:
Will do with general clean up post Dev->Release branch. Thx.